mne/viz/evoked.py

```diff
@@ -71,12 +71,13 @@ def _butterfly_on_button_press(event, params):
     params['need_draw'] = False
 
 
-def _butterfly_onselect(xmin, xmax, ch_types, evoked, text=None):
+def _butterfly_onselect(xmin, xmax, ch_types, info, data, times, text=None,
+                        psd=False):
```

Comment:
_butterfly_onselect -> _line_plot_onselect ?

it's not a butterfly anymore for psd

also

psd -> merge_grad_method = 'mean' | 'rms'